packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -634,6 +634,17 @@ export function unhideTextInstance(
   textInstance.nodeValue = text;
 }
 
+export function clearContainer(container: Container): void {
+  if (container.nodeType === ELEMENT_NODE) {
+    ((container: any): Element).innerHTML = '';
```

Comment:
I believe a faster way is to do:

`container.textContent = ''`

Reply:
Huh. That's interesting to note.

```js
document.documentElement.textContent = '';
// results in:
// <html></html>

document.documentElement.innerHTML = '';
// results in:
// <html>
//    <head></head>
//    <body></body>
// </html>
```